### DISCUSSION TITLE
Backtest Results

### DISCUSSION THREAD

**Discussion by mujahidazamcsm** [conor19w/Binance-Futures-Trading-Bot#13] [General]
Hi @conor19w 

Can we plz have the backtest results for various strategies/timeframes listed here is discussions or on a separate section.

**Comment by conor19w** [comment]
I don't have much data downloaded, I'll download some today but you should have a look yourself too, If I do anything it'll probably just be the last 2 months of data on the common time frames. Which wouldn't be an in depth look at the strategies just a starting point.

**Reply by mujahidazamcsm** [reply]
sure.. if you already have any results plz share the same. I will also start with backtesting and upload the results here. This will help anyone else that might look for it which can be used as reference
